docs/docsource/operation/Zenko_CLI/managing_bucket_policies.rst

```diff
@@ -0,0 +1,18 @@
+.. _managing bucket policies:
+
+Managing Bucket Policies
+========================
+
+In addition to controlling access using access control list (ACL) policies
+(user-based), bucket owners can control access at the resource level using
+cross-origin resource sharing (CORS) policies and bucket policies. Using bucket
+policies, a bucket's owner can set policies that apply to all objects in a
```

Comment:
Policies can apply to the bucket itself or only specific objects within a bucket, too, not sure if that's worth noting

Reply:
noted...